platform/jobs/edxPlatformQualityPr.groovy

```diff
@@ -56,6 +56,48 @@ Map publicJobConfig = [
     diffJob: 'edx-platform-quality-diff'
 ]
 
+Map django19JobConfig = [
+    open : true,
+    jobName : 'edx-platform-django-1.9-quality-flow-pr',
+    subsetJob: 'edx-platform-test-subset',
+    repoName: 'edx-platform',
+    workerLabel: 'django-upgrade-worker',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/django-1.9/quality',
+    triggerPhrase: /.*jenkins\W+run\W+django19\W+quality.*/,
+    defaultTestengBranch: 'master',
+    commentOnly: true,
+    djangoVersion: '1.9'
+]
+
+Map django110JobConfig = [
+    open : true,
+    jobName : 'edx-platform-django-1.10-quality-flow-pr',
+    subsetJob: 'edx-platform-test-subset',
+    repoName: 'edx-platform',
+    workerLabel: 'django-upgrade-worker',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/django-1.10/quality',
+    triggerPhrase: /.*jenkins\W+run\W+django110\W+quality.*/,
+    defaultTestengBranch: 'master',
+    commentOnly: true,
+    djangoVersion: '1.10'
+]
+
+Map django111JobConfig = [
```

Comment:
Should we create a special flow-worker, similar to what you did here: https://github.com/edx/jenkins-job-dsl/pull/312 ?

Reply:
Not a bad idea. since we're just running them by comment lets keep it as is for now.